src/import/ImportRaw.cpp

```diff
@@ -436,11 +423,20 @@ ImportRawDialog::ImportRawDialog(wxWindow * parent,
       }
       S.EndMultiColumn();
 
-      //
-      // Preview Pane goes here
-      //
+      S.SetBorder(5);
+      S.StartTwoColumn();
+      {
+         /* i18n-hint: Guess format of raw file */
+         S.Id(DetectID).AddButton(XXO("Detect"));
+
+         //
+         // Preview Pane goes here
+         //
+
+         S.AddStandardButtons();
```

Comment:
This might be done better by passing more arguments to AddStandardButtons, and avoiding need for the TwoColumn.

Pass `eOkButton | eCancelButton | eApplyButton` and do like below to change the name of the Apply button.  Then bind OnDetect to the id `wxID_APPLY`.

Reply:
This changes button order from Detect, Import, Cancel to Import, Cancel, Detect. And default focus on Detect button.
I'm not going to apply this.

Reply:
And I think using apply button for detect is idiomatically wrong...

Reply:
The "AddStandardButtons" function should be reworked to be more flexible, to cover cases like this.  But sure, that's out of scope for this.